<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetSoundAudio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>20</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub01:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Main:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub02:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub03:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub04:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub05:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub06:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Sub Nob</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>01</string>
     </property>
    </widget>
    <widget class="QDial" name="dial_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>51</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>40</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>51</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_5">
     <property name="geometry">
      <rect>
       <x>227</x>
       <y>100</y>
       <width>51</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_4">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>293</x>
       <y>40</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>51</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>393</x>
       <y>40</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_7">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>51</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>493</x>
       <y>40</y>
       <width>20</width>
       <height>231</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_8">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>51</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_7">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>81</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>03</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>138</x>
       <y>80</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>02 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>348</x>
       <y>81</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>04</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>448</x>
       <y>81</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>05</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>549</x>
       <y>81</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>06</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Main Nob</string>
    </property>
    <widget class="QDial" name="dial">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>141</width>
       <height>201</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAuthor"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuClients">
     <property name="title">
      <string>Clients</string>
     </property>
     <addaction name="actionSub01"/>
     <addaction name="actionSub02"/>
     <addaction name="actionSub03"/>
     <addaction name="actionSub"/>
     <addaction name="actionSub05"/>
     <addaction name="actionSub06"/>
    </widget>
    <addaction name="actionServer_Port"/>
    <addaction name="menuClients"/>
    <addaction name="actionRest_All_Subs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionServer_Port">
   <property name="text">
    <string>Server Port</string>
   </property>
  </action>
  <action name="actionSub01">
   <property name="text">
    <string>sub01</string>
   </property>
  </action>
  <action name="actionSub02">
   <property name="text">
    <string>sub02</string>
   </property>
  </action>
  <action name="actionSub03">
   <property name="text">
    <string>sub03</string>
   </property>
  </action>
  <action name="actionSub">
   <property name="text">
    <string>sub04</string>
   </property>
  </action>
  <action name="actionSub05">
   <property name="text">
    <string>sub05</string>
   </property>
  </action>
  <action name="actionSub06">
   <property name="text">
    <string>sub06</string>
   </property>
  </action>
  <action name="actionRest_All_Subs">
   <property name="text">
    <string>Rest All Subs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dial</tabstop>
  <tabstop>verticalSlider</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>spinBox_10</tabstop>
  <tabstop>spinBox_11</tabstop>
  <tabstop>dial_2</tabstop>
  <tabstop>verticalSlider_3</tabstop>
  <tabstop>dial_3</tabstop>
  <tabstop>verticalSlider_4</tabstop>
  <tabstop>dial_4</tabstop>
  <tabstop>verticalSlider_5</tabstop>
  <tabstop>dial_5</tabstop>
  <tabstop>verticalSlider_6</tabstop>
  <tabstop>dial_6</tabstop>
  <tabstop>verticalSlider_7</tabstop>
  <tabstop>dial_7</tabstop>
  <tabstop>verticalSlider_8</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_7</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_11</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
